governance/votes/Executive vote - April 1, 2022.md

```diff
@@ -0,0 +1,87 @@
+
+### Core Unit Budget Streams
+
+As per multiple successful budget proposals (linked below), a total of **7,945,149.9 DAI** will be streamed to multiple Core Units if this executive proposal passes:
+
+- 1,079,793 DAI will be streamed to the [GovAlpha Core Unit (GOV-001)](https://mips.makerdao.com/mips/details/MIP39c2SP3) at 0x01D26f8c5cC009868A4BF66E268c17B057fF7A73 starting on 2022-04-01 and ending on 2023-04-01 as a result of [their most recent budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP59).
+- 700,356.90 DAI will be streamed to the [Immunefi Security Core Unit (IS-001)](https://mips.makerdao.com/mips/details/MIP39c2SP24) at 0xd1F2eEf8576736C1EbA36920B957cd2aF07280F4 starting on 2022-03-01 and ending on 2022-08-01 as a result of [an amendment to their MIP40c3-SP42 budget proposal](https://mips.makerdao.com/mips/details/MIP4c2SP17).
+- 6,165,000 DAI will be streamed to the [Real-World Finance Core Unit (RWF-001)](https://mips.makerdao.com/mips/details/MIP39c2SP1) at 0x96d7b01Cc25B141520C717fa369844d34FF116ec starting on 2022-04-01 and ending on 2022-12-31 as a result of [their morst recent budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP61).
+	- The existing DAI budget stream for this Core Unit will be revoked.
+
+### Core Unit MKR Vesting Streams
+
+As per their successful [MKR budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP60), a total of **167.64 MKR** will be vested to members of the [GovAlpha Core Unit (GOV-001)](https://mips.makerdao.com/mips/details/MIP39c2SP3) if this executive proposal passes, as itemized below:
+
+- **73.70 MKR** will be streamed to Facilitator Long For Wisdom at 0xC818Ae5f27B76b4902468C6B02Fd7a089F12c07b starting on 2022-02-08 and ending on 2023/02/08 with a cliff date of 2023-02-08.
+- **52.74 MKR** will be streamed to Facilitator prose11 at 0xbfDD0E744723192f7880493b66501253C34e1241 starting on 2022-02-08 and ending on 2023/02/08 with a cliff date of 2023-02-08.
+- **41.20 MKR** will be streamed to the GovAlpha multisig at 0x01D26f8c5cC009868A4BF66E268c17B057fF7A73 starting on 2022-02-08 and ending on 2023/02/08 with a cliff date of 2023-02-08.
+
+### Retroactive MKR Compensation for Long For Wisdom
+
+As per [GovAlpha Core Unit (GOV-001)](https://mips.makerdao.com/mips/details/MIP39c2SP3) successful [MKR budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP60), a total of **60 MKR** will be transferred to Facilitator Long For Wisdom at  0xC818Ae5f27B76b4902468C6B02Fd7a089F12c07b if this executive proposal passes.
+
+### Gelato Keeper Network DAI Budget Stream
+
+As per the ratified [MIP63c4-SP3: Onboard Keeper Network - Gelato Network](https://mips.makerdao.com/mips/details/MIP63c4SP3), a total of **731,000 DAI** will be streamed to Gelato Keeper Network if this executive proposal passes:
+
+- **731,000 DAI** will be streamed to Gelato Keeper Network at 0x926c21602fec84d6d0fa6450b40edba595b5c6e4 starting on 2022-04-01 and ending on 2024-03-31.
```

Comment:
Can you include the details of the buffer max and buffer min here as well?
maybe also that it's 1000 DAI/day since that was what the MIP explicitly approved